src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -289,10 +299,51 @@ public static int getFuelValue(ItemStack itemStack)
         return fuelValue;
     }
 
+    /**
+     * Links a set of parts to an {@link IPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IPartFactory factory, ResourceLocation... names)
+    {
+        MultipartRegistry.registerPartFactory(factory, names);
+    }
+
+    /**
+     * Links a set of parts to an {@link IAdvancedPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IAdvancedPartFactory factory, ResourceLocation... names)
+    {
+        MultipartRegistry.registerPartFactory(factory, names);
+    }
+
+    /**
+     * Registers a part along with an identifier. A default part factory is automatically created.
+     */
+    public static void registerPart(Class<? extends IMultipart> clazz, ResourceLocation name)
+    {
+        MultipartRegistry.registerPart(clazz, name);
+    }
+
+    /**
+     * Registers an {@link IPartConverter}.
+     */
+    public static void registerPartConverter(IPartConverter converter)
+    {
+        MultipartRegistry.registerPartConverter(converter);
+    }
+
+    /**
+     * Registers an {@link IReversePartConverter}.
+     */
+    public static void registerReversePartConverter(IReversePartConverter converter)
```

Comment:
None of these functions should exist... Everything should go through the original register(RegistryEntry) function.